test/test_sparse.py

```diff
@@ -1694,6 +1694,184 @@ def test_log1p(self):
+            device=self.device
+        )
+        self._test_abs_absolute(input_coalesced, torch.tensor([3.0, -4.0, 5.0]))
+
+        # hybrid sparse input
+        input_coalesced = torch.sparse_coo_tensor(
+            indices=torch.tensor([[1, 3], [2, 4]]),
+            values=torch.tensor([[-1.0, 3.0], [-5.0, 7.0]]),
+            size=[4, 5, 2],
+            device=self.device
+        )
+        true_dense = torch.sparse_coo_tensor(
+            indices=torch.tensor([[1, 3], [2, 4]]),
+            values=torch.tensor([[1.0, 3.0], [5.0, 7.0]]),
+            size=[4, 5, 2],
+            device=self.device
+        ).to_dense()
+        self._test_abs_absolute(input_coalesced, true_dense)
+
+        if self.is_uncoalesced:
+            # test uncoalesced input
+            input_uncoalesced = torch.sparse_coo_tensor(
+                indices=torch.tensor([[0], [1], [2], [0], [1], [2]]).transpose(1, 0),
+                values=torch.tensor([2.0, -3.0, -4.0, 1.0, -1.0, 1.5]),
+                size=[3, ],
+                device=self.device
+            )
+            self._test_abs_absolute(input_uncoalesced, torch.tensor([3.0, 4.0, 2.5]))
+
+            # test on empty sparse tensor
+            input_uncoalesced = torch.sparse_coo_tensor(
+                indices=torch.zeros([2, 0]),
+                values=torch.zeros([0, 5, 5, 5, 5, 5, 5, 0]),
+                size=[0, 0, 5, 5, 5, 5, 5, 5, 0],
+                device=self.device
+            )
+            self._test_abs_absolute(input_uncoalesced, torch.zeros([0, 0, 5, 5, 5, 5, 5, 5, 0]))
+
+            # test on empty sparse tensor, values empty, index not empty
+            # crashes with RuntimeError: CUDA error: an illegal memory access was encountered
+            # Exception raised from copy_kernel_cuda at pytorch/aten/src/ATen/native/cuda/Copy.cu:200
+            # input_uncoalesced = torch.sparse_coo_tensor(
+            #     indices=torch.zeros([1, 5]),
+            #     values=torch.zeros([5, 6, 0]),
+            #     size=[5, 6, 0],
+            #     device=self.device
+            # )
+            # self._test_abs(input_uncoalesced, torch.zeros([5, 6, 0]))
+
+    def _test_sign(self, sparse_tensor, dense_tensor):
+        expected_output = dense_tensor.sign()
+        self.assertEqual(expected_output, torch.sign(sparse_tensor).to_dense())
+        self.assertEqual(expected_output, sparse_tensor.sign().to_dense())
+        self.assertEqual(expected_output, sparse_tensor.coalesce().sign_().to_dense())
+
+        # test in-place op on uncoalesced input
+        with self.assertRaisesRegex(RuntimeError, "in-place on uncoalesced tensors is not supported"):
+            sparse_tensor.sign_()
+
+        # TODO: vfdev-5: Currently backprop through sign op on sparse tensor is not working due to
+        # RuntimeError: memory format option is only supported by strided tensors
+        # Exception raised from empty_like at pytorch/aten/src/ATen/native/TensorFactories.cpp:288
```

Comment:
Consider this as a blocker. Could you address the empty_like issue separately from this PR?

Reply:
> Could you address the empty_like issue separately from this PR?

Done, #43699